### ISSUE TITLE
RFE: DNS-based discovery via TXT records

### ISSUE THREAD

**Issue by glpatcern** [cs3org/OCM-API#141]
Following the discussion in #134, there is some consensus that a DNS-based discovery via TXT records is a useful add-on to the protocol and can provide an alternative to the Invitation flow.

However the idea is to keep it for a future 2.0 version of the protocol, as to make it useful it would need to be enforced, which is a breaking change with respect to the current 1.x implementations.

This issue is then to keep track of that decision.

**Comment by mickenordin** [comment]
I agree that it would be useful, not least because we would be able to make any email-address into a valid OCM-address no matter on what subdomain the OCM-server is deployed.

**Comment by KrausMatthias** [comment]
One might also think about some webfinger based mechanism as used by Mastodon? https://docs.joinmastodon.org/spec/webfinger/#mastodons-requirements-for-webfinger
